deepmd/pt/utils/tabulate.py

```
@@ -0,0 +1,920 @@
+    if functype == 1:
+        return -2 * y * (1 - y * y)
+    elif functype == 2:
+        var1 = np.tanh(SQRT_2_PI * (xbar + GGELU * xbar ** 3))
+        var2 = SQRT_2_PI * (1 - var1 ** 2) * (3 * GGELU * xbar ** 2 + 1)
+        return 3 * GGELU * SQRT_2_PI * xbar ** 2 * (1 - var1 ** 2) \
+            - SQRT_2_PI * xbar * var2 * (3 * GGELU * xbar ** 2 + 1) * var1 \
+            + var2
+    elif functype == 3:
+        return 0
+    elif functype == 4:
+        return 0
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)